packages/css-library/tokens/color.json

```diff
@@ -1,65 +1,65 @@
 {
   "color": {
-    "base": { "value": "#212121"},
-    "white": { "value": "#ffffff"},
-    "black": { "value": "{uswds-system-color-gray-100}"},
-    "link-default": { "value": "#004795"},
-    "link-visited": { "value": "#4c2c92"},
-    "warning-message": { "value": "#fac922"},
-    "feedback-warning-background": { "value": "#fac922"},
+    "base": { "value": "#212121" },
+    "white": { "value": "#ffffff" },
+    "black": { "value": "{uswds-system-color-gray-100}" },
+    "link-default": { "value": "{uswds-system-color-blue-vivid-60}" },
+    "link-visited": { "value": "{uswds-system-color-violet-vivid-70}" },
+    "warning-message": { "value": "#fac922" },
+    "feedback-warning-background": { "value": "#fac922" },
     "primary": {
-      "*": { "value": "#0071bb"},
-      "darker": { "value": "#003e73"},
-      "darkest": { "value": "#112e51"},
+      "*": { "value": "{uswds-system-color-blue-vivid-60}" },
+      "darker": { "value": "{uswds-system-color-blue-warm-vivid-70}" },
```

Comment:
From Matt in regards to Issue #2228, primary-darkest should go away, primary-darker should become "uswds-system-color-blue-warm-vivid-80", and there should be a new color "primary-dark" set to "uswds-system-color-blue-warm-vivid-70".

Reply:
Fixed! Confirmed they're showing up in storybook as well: 

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/83d6be3a-ea4a-4d82-957a-4d944e6b6044)